Andrewz

andrew123zhang opened the pull request:
first pull request/merge

andrew123zhang committed:
Wrote a basic python program to upload & sync ~/bioda/database with s3://andrew-zhang-backup-bucket.
Added a comment with a similar line to download & sync the above, for future use.
-Andrew Zhang

andrew123zhang committed:
last commit before code review
-Andrew Zhang

glicxu commented on src/exampleS3Upload.py:
```diff
@@ -0,0 +1,7 @@
+import os
+
```
please use logging to log the start, end, and also status of s3 sync

andrew123zhang committed:
Added logging, removed hardcoding in favor of variables and fstrings.
Added some try/except statements, not sure if they are properly set up; not sure which Exception to choose
Still need to consolidate variables in config.

-Andrew Zhang
